packages/size-limit/read-pkg-up.js

```diff
@@ -0,0 +1,42 @@
+const path = require('path')
```

Comment:
Can you import methods like `let { resolve } = require(…)` to keep consistency with other files

Reply:
Fixed in 84789586cb9d4aa93cb6945325a4d26933dce720